app/assets/javascripts/discourse/app/templates/modal/user-status.hbs

```diff
@@ -0,0 +1,24 @@
+{{#d-modal-body}}
+  {{#conditional-loading-spinner condition=loading}}
+    <div class="control-group user-status-description-wrap">
+      {{input
```

Comment:
We don't enforce it yet, but ftr correct indentation is:

```
{{input
  foo=1
  bar="baz"
}}
```

Reply:
☑️